src/proof/nova.rs

```diff
@@ -866,7 +928,7 @@ pub mod tests {
         let s = &mut Store::<Fr>::default();
         let expected = s.num(3);
         let terminal = s.get_cont_terminal();
-        test_aux::<Coproc<Fr>>(
+        test_aux::<_, _, C1<'_, _, Coproc<_>>>(
```

Comment:
Can we abbreviate this type with a local type definition? It will probably avoid a bunch of line additions.

It would also setup an organized pattern for when we introduce the LEM tests.

Reply:
I'm not quite sure what local type definition you have in mind, but I'm of course in favor of the PR re-organizing the proof tests to put LEM & Lurk side-by-side doing some heavy refactoring to do so. 

The line changes made here are such that those generic methods that need annotation now bear the exact same annotation on each line, syntactically - this makes them ripe for mechanized refactoring. So I'm not too worried about that.

Reply:
https://github.com/huitseeker/lurk-rs/pull/8 for a -800 lines diff